### PR TITLE
Russian Regions

### PR DESCRIPTION
**Pull request by luckyraul**
### Description (*)
Add current Russian Federation regions with their codes according to http://www.gov.ru/

### Manual testing scenarios (*)
1. Setup DB
2. Check Regions in Region field for Russia

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

File: app/code/Magento/Directory/Setup/Patch/Data/AddDataForRussia.php [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+    {
+        return [];
+    }
+
+    /**
+     * Poland states data.
+     *
+     * @return array
+     */
+    private function getDataForPoland()
```
Comment (slavvka) [inline app/code/Magento/Directory/Setup/Patch/Data/AddDataForRussia.php]:
```suggestion
    private function getDataForRussia()
```

File: app/code/Magento/Directory/Setup/Patch/Data/AddDataForRussia.php [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+    /**
+     * @inheritdoc
+     */
+    public function apply()
+    {
+        /** @var DataInstaller $dataInstaller */
+        $dataInstaller = $this->dataInstallerFactory->create();
+        $dataInstaller->addCountryRegions(
+            $this->moduleDataSetup->getConnection(),
+            $this->getDataForPoland()
```
Comment (slavvka) [inline app/code/Magento/Directory/Setup/Patch/Data/AddDataForRussia.php]:
```suggestion
            $this->getDataForRussia()
```

File: app/code/Magento/Directory/Setup/Patch/Data/AddDataForRussia.php [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+            ['RU', '01', 'Республика Адыгея (Адыгея)'],
+            ['RU', '04', 'Республика Алтай'],
+            ['RU', '02', 'Республика Башкортостан'],
+            ['RU', '03', 'Республика Бурятия'],
+            ['RU', '05', 'Республика Дагестан'],
+            ['RU', '06', 'Республика Ингушетия'],
+            ['RU', '08', 'Республика Калмыкия'],
+            ['RU', '10', 'Республика Карелия'],
+            ['RU', '11', 'Республика Коми'],
+            ['RU', '91', 'Республика Крым'],
```
Comment (oshmyheliuk) [inline app/code/Magento/Directory/Setup/Patch/Data/AddDataForRussia.php]: This is not Russian territory.
```suggestion
```

File: app/code/Magento/Directory/Setup/Patch/Data/AddDataForRussia.php [hunk 4]
```diff
@@ -0,0 +1,168 @@
...
```
Comment (oshmyheliuk) [inline app/code/Magento/Directory/Setup/Patch/Data/AddDataForRussia.php]: This is not Russian territory.
```suggestion
```

**Comment by slavvka** [comment]
@luckyraul Please provide meaningful description according to the template

File: app/code/Magento/Directory/Setup/Patch/Data/AddDataForRussia.php [hunk 5]
```diff
@@ -0,0 +1,168 @@
...
+     */
+    public function apply()
+    {
+        /** @var DataInstaller $dataInstaller */
+        $dataInstaller = $this->dataInstallerFactory->create();
+        $dataInstaller->addCountryRegions(
+            $this->moduleDataSetup->getConnection(),
+            $this->getDataForRussia()
+        );
+    }
```
Comment (slavvka) [inline app/code/Magento/Directory/Setup/Patch/Data/AddDataForRussia.php]:
```suggestion
        return $this;
    }
```

Comment (luckyraul) [reply app/code/Magento/Directory/Setup/Patch/Data/AddDataForRussia.php]: AddDataForAustralia - no return $this;
AddDataForBelgium  - no return $this;
AddDataForChina  - no return $this;
AddDataForColombia  - no return $this;
.....

Comment (slavvka) [reply app/code/Magento/Directory/Setup/Patch/Data/AddDataForRussia.php]: @luckyraul We are not in the kindergarten to refer to other bad examples. You have the failing test so please make it green

File: app/code/Magento/Directory/Setup/Patch/Data/AddDataForRussia.php [hunk 6]
```diff
@@ -0,0 +1,170 @@
...
+            ['RU', '01', 'Республика Адыгея (Адыгея)'],
+            ['RU', '04', 'Республика Алтай'],
+            ['RU', '02', 'Республика Башкортостан'],
+            ['RU', '03', 'Республика Бурятия'],
+            ['RU', '05', 'Республика Дагестан'],
+            ['RU', '06', 'Республика Ингушетия'],
+            ['RU', '08', 'Республика Калмыкия'],
+            ['RU', '10', 'Республика Карелия'],
+            ['RU', '11', 'Республика Коми'],
+            ['RU', '91', 'Республика Крым'],
```
Comment (zamoroka) [inline app/code/Magento/Directory/Setup/Patch/Data/AddDataForRussia.php]: The Republic of Crimea is officially part of Ukraine

File: app/code/Magento/Directory/Setup/Patch/Data/AddDataForRussia.php [hunk 7]
```diff
@@ -0,0 +1,170 @@
...
+            ['RU', '16', 'Республика Татарстан (Татарстан)'],
+            ['RU', '17', 'Республика Тыва'],
+            ['RU', '19', 'Республика Хакасия'],
+            ['RU', '61', 'Ростовская область'],
+            ['RU', '62', 'Рязанская область'],
+            ['RU', '63', 'Самарская область'],
+            ['RU', '64', 'Саратовская область'],
+            ['RU', '65', 'Сахалинская область'],
+            ['RU', '66', 'Свердловская область'],
+            ['RU', '92', 'Севастополь'],
```
Comment (zamoroka) [inline app/code/Magento/Directory/Setup/Patch/Data/AddDataForRussia.php]: This is the part of Ukraine

**Comment by zamoroka** [comment]
The Republic of Crimea and Sevastopol are officially part of Ukraine

**Pull request closed without merging**